README.md

```diff
@@ -4,14 +4,31 @@
 
 The main goal is to put into practice the main concepts of Object Oriented Programming in Ruby programming language. Particularly, classes and access to their attributes.
 
-![image](https://user-images.githubusercontent.com/55923773/76424191-decda800-63e2-11ea-9f7d-3b2fa04f0ddb.png)
+![image](https://user-images.githubusercontent.com/55923773/76505809-070de300-6485-11ea-9e0a-cb20a3094f1b.png)
 
 
 ## Built With
 
 - Ruby
 - Rubocop
 
+### Project Structure
+
+```
+├── README.md
+├── .rubocop.yml
+├── .stickler.yml
+├── bin
+│   └── main.rb
+└── lib
+    └── board.rb
+    └── game.rb
+    └── player.rb
+```
+### Game Instructions
+
+Its a two player game. Each player can select a token - `X`, `O`. The first player gets the `X` token and `O` is reserved for the second player.
+
```

Comment:
Let's add a more detailed game instruction. We can also add a wiki link of tic-tac-toe or/and screenshots of the step taken to play the game till there's a win or draw.

Instructions of what wins, losses or draws the game should be added.

Reply:
Done.